relay_hub/server.py

```
@@ -0,0 +1,330 @@
+            self.auth.ensure_can_consume(actor, message.meta.to)
+            return message_to_dict(message, include_content=True)
+
+    def _register_resources(self) -> None:
+        mcp = self.mcp
+
+        @mcp.resource("relay://mailbox/{mailbox}/{message_id}")
+        def mailbox_resource(mailbox: str, message_id: str) -> Dict[str, Any]:
+            request = FetchRequest(id=message_id, include_ack=True)
+            message = self.storage.fetch(request)
+            if not message or message.meta.to != mailbox:
+                raise RuntimeError("message not found")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)